### ISSUE TITLE
Trash can icon in patient self-reg needs a fix

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#3986]
## Area or Component
Patient Experience - Self registration

## Quick Summary
There's a trash can icon for emails in patient self-registration (and the add/edit patient flows.) This icon is hidden from screenreaders, but does provide occasional functionality if an email has been input.

The larger issue here is that the button behavior is inconsistent. If the field is blank and you click the button, nothing happens. If the field is partially filled out, it clears. If there are multiple email fields, it allows you to remove the field entirely.

## Accessibility Violation
Two:
aria-hidden="true" is used on informative content making it inaccessible to assistive technology. https://axe.deque.com/rule-help/en/semantic-hidden
The text alternative is missing. https://axe.deque.com/rule-help/en/alt-text-missing

## Action Requested
Needs design input, but my guess is that we should change the behavior of this icon or remove it (display=none) until there's more than one email address. 

## Screencast
![SimpleReport (6)](https://user-images.githubusercontent.com/80282552/179129115-25411750-abb8-4ffc-9818-764e1652f42d.gif)

## Source Code (optional)
https://github.com/CDCgov/prime-simplereport/blob/c3e6b2dc330021e425e01657eaee613c51f9e11c/frontend/src/app/patients/Components/PersonForm.tsx

Note: This is likely going to be solved at the `PersonForm` level, meaning we also need to check the in-app `Add/Edit Person` page since the component is shared.

## Original Deque Issue
semantic-hidden | aria-hidden="true" is used on informative content making it inaccessible to assistive technology. | Trash can icon has aria-hidden true for emails. The icon is actually not usable for the first email entry, so the preferred solution would be to take it off entirely vs having an incorrect icon/label | aria-hidden="true" is used incorrectly | https://axe.deque.com/rule-help/en/semantic-hidden | critical | Yes | No | Images | ["body > div#root:nth-of-type(1) > div.bg-base-lightest.minh-viewport:nth-of-type(1) > div.bg-base-lightest:nth-of-type(3) > div#registration-container.grid-container.maxw-tablet.padding-y-3:nth-of-type(1) > div.prime-formgroup:nth-of-type(2) > fieldset.usa-fieldset:nth-of-type(1) > div.usa-form:nth-of-type(2) > div.usa-form:nth-of-type(1) > div:nth-of-type(1) > div.display-flex:nth-of-type(1) > div.flex-align-self-end:nth-of-type(2) > button.usa-button--unstyled.padding-105.height-5:nth-of-type(1) > svg.svg-inline--fa.fa-trash.fa-w-14.text-error:nth-of-type(1)"] | aria-hidden="true" is used on informative content making it inaccessible to assistive technology. | <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="trash" class="svg-inline--fa fa-trash fa-w-14 text-error" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"> | wcag2a,wcag131a,wcag132a | rnv7@cdc.gov | Patient Self-Registration - Form | https://stg.simplereport.gov/app/register/9C7BB |   | 2022-07-13T22:23:43.940Z
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
alt-text-missing | The text alternative is missing. | No alt text for the trash can icon | Text alternative is missing | https://axe.deque.com/rule-help/en/alt-text-missing | critical | Yes | No | Images | ["body > div#root:nth-of-type(1) > div.bg-base-lightest.minh-viewport:nth-of-type(1) > div.bg-base-lightest:nth-of-type(3) > div#registration-container.grid-container.maxw-tablet.padding-y-3:nth-of-type(1) > div.prime-formgroup:nth-of-type(2) > fieldset.usa-fieldset:nth-of-type(1) > div.usa-form:nth-of-type(2) > div.usa-form:nth-of-type(1) > div:nth-of-type(1) > div.display-flex:nth-of-type(1) > div.flex-align-self-end:nth-of-type(2) > button.usa-button--unstyled.padding-105.height-5:nth-of-type(1) > svg.svg-inline--fa.fa-trash.fa-w-14.text-error:nth-of-type(1)"] | The text alternative is missing. | <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="trash" class="svg-inline--fa fa-trash fa-w-14 text-error" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"> | wcag2a,wcag111a,wcag111b,wcag111f | rnv7@cdc.gov | Patient Self-Registration - Form | https://stg.simplereport.gov/app/register/9C7BB |   | 2022-07-13T22:23:43.969Z

**Comment by kenieh** [comment]
Discussed this ticket during sprint planning.

The design action going forward is to remove the trash can icon and only include the trash icon for the additional fields when a user clicks "+ Add another email". Doing so will mirror the functionality and design of the field above for the Phone number fields. None of the other form fields here have a trash can icon, so this will also build consistency across the form.

@vz3 with that, this issue is resolved.
